Services/Mail/test/ilMailAddressParserTest.php

```diff
@@ -75,10 +81,10 @@ public function emailInvalidAddressesProvider() : array
     {
         return [
             'Trailing Dot in Local Part' => [
-                'phpunit.'
+                'phpunit@'
             ],
             'Trailing Dot in Local Part of Email Address' => [
-                'phpunit.@ilias.de'
+                'phpunit@@ilias.de'
```

Comment:
The `data set` should be renamed now, because now there is a trailing `@`.

Reply:
Added Quotes (") instead of At (@) cause this are filtered earlier